scripts/generate-coverage.sh

```diff
@@ -4,12 +4,20 @@
 # go test can't generate code coverage for multiple packages in one command
 
 set -e
+ARCH=$(uname -m)
 echo "" > coverage.txt
-go test -i -race ./cmd/odo
+if [ "${ARCH}" == "s390x" ]; then
+    go test -i ./cmd/odo
```

Comment:
please add the comment why you are disabling `-race` flag here. As I can see in [comment](https://github.com/openshift/odo/issues/3756#issuecomment-674852273) its not supported so would suggest you to update it here as well

Reply:
@prietyc123 Done. Thanks.